include/core/CPersistUtils.h

```diff
@@ -271,51 +253,50 @@ class CORE_EXPORT CPersistUtils {
         char m_PairDelimiter;
     };
 
-    //! \brief Utility to convert a string to a built in type
-    //! using CStringUtils functions.
+    //! \brief Converts a string to a built in type using CStringUtils functions.
     class CORE_EXPORT CBuiltinFromString {
     public:
         CBuiltinFromString(const char pairDelimiter)
             : m_PairDelimiter(pairDelimiter) {
-            m_Token.reserve(15);
+            m_Token.reserve(17);
```

Comment:
Similar to what I said in the other comment, it might be best not to `reserve` here in the modern world.

Reply:
Changed in [fa3ab02](https://github.com/elastic/ml-cpp/pull/1274/commits/fa3ab02793b790aa31efb0c0cd8524a645f0db6a).